clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -2751,6 +2751,13 @@ void CodeGenModule::SetLLVMFunctionAttributesForDefinition(const Decl *D,
       B.addAttribute("aarch64_new_zt0");
   }
 
+  // Handle flatten_depth attribute for depth-based inlining
+  if (const FlattenDepthAttr *FDA = D->getAttr<FlattenDepthAttr>()) {
+    // Add the flatten_depth attribute with the max depth value as a typed int
```

Comment:
Move the comment outside of the 'if', and remove the curley braces.  The two variants of the comment are redundant anyway.